src/frontend/efiling-frontend/package.json

```diff
@@ -15,13 +15,15 @@
     "history": "^5.0.0",
     "jsonwebtoken": "^8.5.1",
     "keycloak-js": "^7.0.1",
+    "moment": "^2.29.1",
```

Comment:
Ah moment 🤣

Reply:
Yeah, hope that's okay. It was the easiest way to parse and format a date to a custom string.